README.md

```diff
@@ -83,6 +83,19 @@ earlier):
 
     sudo -u postgres psql bookbrainz -c "SELECT name FROM bookbrainz.editor"
 
+You are also required to set the password of PostgreSQL for your local.
+you can do this by
+
+
+    sudo -u postgres psql
+    psql (9.6.9)
+    Type "help" for help.
```

Comment:
I would remove the two lines `psql (9.6.9)` and `Type "help" for help.` to limit possible confusion.